pages/k8s/install-manual.md

```diff
@@ -148,110 +246,25 @@ versions of the **CDK** bundle are shown in the table below:
-integration, continue reading below first.
-
-
-### Additional configuration
-
-To allow Kubernetes to access resources and functionality of the underlying
-cloud upon which it is deployed, additional integrator charms are available. When
-installing with **conjure-up**, these charms are automatically added to the
-deployment and configured appropriately.
-
-Adding the integrator charms directly with **Juju** is not recommended - it is more
-reliable (and easier) to include these charms at deployment time, using the "overlay"
-method as [described below](#overlay). The manual deployment steps shown here are
-only for reference to give a better understanding of how these charms work in relation to
-**CDK**.
-
-This table explains which charms are
-used:
-
-| Cloud     | Integrator charm   |  Juju deploy command  | Notes/docs  |
-|------------|----------------------|-------------------------------------------------|-----------------------------------------------------------|
-| AWS        | aws-integrator       | juju deploy cs:~containers/aws-integrator       | [docs][aws-docs]      |
-| Azure      | azure-integrator     | juju deploy cs:~containers/azure-integrator     | [docs](https://jujucharms.com/u/containers/azure-integrator/)     |
-| Google     | gcp-integrator       | juju deploy cs:~containers/gcp-integrator       | [docs][gcp-docs]      |
-| Local      | N/A                  | N/A                                             |                                                           |
-| OpenStack  | openstack-integrator | juju deploy cs:~containers/openstack-integrator | [docs](https://jujucharms.com/u/containers/openstack-integrator/) |
-| Rackspace  | openstack-integrator | juju deploy cs:~containers/openstack-integrator | [docs](https://jujucharms.com/u/containers/openstack-integrator/) |
-| vSphere    | vsphere-integrator   | juju deploy cs:~containers/vsphere-integrator   | [docs](https://jujucharms.com/u/containers/vsphere-integrator/ )  |
-
-The charm should be deployed and relationships established with both the
-`kubernetes-worker` and `kubernetes-master` charms. For example, in the case of
-AWS:
-
-```bash
-juju deploy cs:~containers/aws-integrator
-juju trust aws-integrator
-juju add-relation aws-integrator kubernetes-master
-juju add-relation aws-integrator kubernetes-worker
-```
-
-The `juju trust` command allows the aws-integrator to make use of the
-credentials stored by **Juju**.
-
-This demonstrates how the charm relates to the rest of the **CDK** bundle, but it is
-recommended to use the [overlay](#overlay) method for installing in practice.
 
 ## Customising the bundle install
 
 A number of the scenarios outlined at the start of this document involved
-customising  the **CDK** install. There are two main ways to do this, using
-overlays or editing the bundle file itself.
+customising  the **Charmed Kubernetes** install. There are two main ways to
+do this, using overlays or editing the bundle file itself.
```

Comment:
this seems awkwardly punctuated. Maybe "There are two main to do this: using overlays and editing the bundle itself" or something like that? Maybe just reword it entirely. "To customize the install one may edit the bundle or use overlays"